x-pack/plugins/screenshotting/server/browsers/chromium/driver.ts

```diff
@@ -35,12 +35,6 @@ export interface ElementPosition {
   };
 }
 
-export interface Viewport {
```

Comment:
This is not needed, and is confusing: `puppeteer.Viewport` is a similar but slightly different interface.